python/tests/util/storage_test.py

```diff
@@ -5,7 +5,41 @@
 import re
 from datetime import datetime
 
-from arcticdb.storage_fixtures.s3 import real_s3_from_environment_variables
```

Comment:
I just made these changes. Why are they being reverted without any meaningful explanation in the commit message?

Reply:
I left a comment, essentially for this part of the code we are relying on installing arcticdb (both the latest version of the package and an older one, currently 3.0.0).
Because your changes are not part of the older version this is breaking the code.
As I have written in the comment, we will reincorporate your changes when they are part of the oldest version that we support.